passkey-api/introduction.mdx

```diff
@@ -0,0 +1,229 @@
+---
+title: Introduction
+description: The Passkey API adds passkey support to your existing auth system.
+---
+
+The Hanko Passkey API adds support for passkeys for **existing users** of your platform. It's designed to be used **together with your own auth system**, and can be used without any other Hanko product.
+
+Users cannot create an account through the Passkey API. Instead, the Passkey API enables your users to add passkeys to their existing account (they have to be logged in first).
+
+<Tip>
+Note that the Passkey API is not designed to be a full authentication solution — it's only intended to be used together with another auth system.
+
+If you need a full, **open-source** auth solution, check out [Hanko Auth](/introduction)
+
+</Tip>
+
+## What it looks like
+
+Most setups look something like the following:
+
+-   your frontend/client the user is using with a device that  
+    [supports the Web Authentication API](https://caniuse.com/webauthn)
+-   your backend that communicates with the Passkey API
+-   the Passkey API, either [self-hosted](https://github.com/teamhanko/passkey-server) or on [Hanko Cloud](https://www.hanko.io/features/hanko-passkey-api)
+
+The Passkey API is designed to be used both from your backend and frontend:
+
+Some requests (e.g. logging in) can be made to the Passkey API directly from the frontend.
+These do not require a secret API key.
+
+All other requests that require a secret API key (e.g. registering a new passkey) should only be made from your backend. You should **never** leak that secret API key to the client.
+
+## Example implementation
+
+This is an example implementation showing how to use the Passkey API with Node.js. (You can use any language you want to, of course!)
+
+However, if you're already using JavaScript/TypeScript for your backend, you can use [\@teamhanko/passkey-backend-sdk](https://www.npmjs.com/package/@teamhanko/passkey-backend-sdk), which do all of the below for you.
+
+Otherwise, please make sure to always send JSON with `Content-Type: application/json`.
+
+As of writing, for the frontend, the Web Authentication API expects you to pass `ArrayBuffer` (instead of plain old objects) in a lot of places, which can be inconvenient.
+
+In the examples below we're using [@github/webauthn-json](https://github.com/github/webauthn-json#githubwebauthn-json) which is a wrapper around the Web Authentication API to make things easier
+
+---
+
+1. **Get your tenant ID**
+
+    The base URL for the Passkey API depends on your `tenant_id`.
+
+    You can [find your tenant ID in your Hanko Cloud account.](https://cloud.hanko.io/organizations)
+
+    [👉 Explanation for tenant ID](faq#tenant-id-and-multitenancy)
```

Comment:
is it intentional to open the link in new tab? We can have it like this `[👉 Explanation for tenant ID](/passkey-api/faq#tenant-id-and-multitenancy)`

Reply:
I didn't know that causes it to open in a new tab. Will fix